xdsl/irdl/irdl.py

```diff
@@ -1513,10 +1634,10 @@ def verify(self, op: Operation):
         irdl_op_verify_arg_list(op, self, VarIRConstruct.RESULT, constraint_vars)
 
         # Verify regions.
-        irdl_op_verify_arg_list(op, self, VarIRConstruct.REGION, constraint_vars)
+        irdl_op_verify_regions(op, self)
 
-        # Verify successors.
-        irdl_op_verify_arg_list(op, self, VarIRConstruct.SUCCESSOR, constraint_vars)
+        # verify successors
```

Comment:
Why is this now in lowercase without a comma, where everything else is?

Reply:
Because strong opinions

Nah, good catch 😋